src/main/java/com/refinedmods/refinedstorage/apiimpl/network/grid/CraftingGridBehavior.java

```diff
@@ -23,8 +24,9 @@
 import java.util.List;
 
 public class CraftingGridBehavior implements ICraftingGridBehavior {
+
     @Override
-    public void onCrafted(INetworkAwareGrid grid, ICraftingRecipe recipe, PlayerEntity player) {
+    public void onCrafted(INetworkAwareGrid grid, ICraftingRecipe recipe, PlayerEntity player, IStackList<ItemStack> networkCraftingItems, IStackList<ItemStack> extractedItems) {
```

Comment:
I don't think this `networkCraftingItems` param is needed, we have a `INetworkAwareGrid`.

Reply:
The networkCraftingItems need to persist through each iteration. It caches the number of items that are in the system. Each iteration I remove some items from that list.